### ISSUE TITLE
No functions are found

### ISSUE THREAD

**Issue by NorteX-dev** [rommelsantor/Timeout#13]
When I try using any of the functions:
`Timeout.set()`, `Timeout.remaining()`, `Timeout.anything()`
it says that the function is undefined.,
![image](https://user-images.githubusercontent.com/32877032/81070977-551ef000-8ee4-11ea-9586-4523c58d865c.png)
Version:
- tried on latest 2.4.0
- tried on 2.3.1
none of which worked.
It's a major bug and I need the functionality that this module provides!

**Comment by NorteX-dev** [comment]
By the way, Timeout variable is specified:
`const Timeout = require("smart-timeout");`

**Comment by NorteX-dev** [comment]
Installed a much lower version & worked... It still doesn't solve the issue, as the latest version is broken.

**Comment by rommelsantor** [comment]
@NorteX-dev Thanks for reporting. That's definitely not good.

In the transition to TypeScript the default export got broken for `require()`. (Fyi, it would have worked for you to try `const Timeout = require('smart-timeout').default`, not that you _should_ have had to do that, of course.)

A fix has been pushed in v2.4.1.

**Comment by NorteX-dev** [comment]
Thanks for the fix.

**Comment by MarcusCalidus** [comment]
This breaks the syntax `import * as Timeout from 'smart-timeout'` in modern Typescript 3.8. You will have to import directly from the lib folder via `import {Timeout} from 'smart-timeout/lib/timeout'`

**Comment by MarcusCalidus** [comment]
the following could be helpful
```
import { Timeout, TimeoutInstance } from './lib/timeout'
export default Timeout
export { Timeout, TimeoutInstance }
```

**Comment by rommelsantor** [comment]
Thanks for the heads-up, @MarcusCalidus!

This is corrected in v2.5.0.
